Islandora/islandora-starter-site
# Include pdf.js library.

# What does this Pull Request do?

Includes pdf.js in the starter site.

This will make the pdf.js library available [edited to say: as part of the starter-site]. Currently when you create an ISLE site that uses the Starter Site, pdf.js is not available. The Playbook does currently install pdf.js on its own, so a PR will follow to remove that.

* **Related GitHub Issue**: Reported by @nigelgbanks 

* **Other Relevant Links**: (Google Groups discussion, related pull requests,
 Release pull requests, etc.)

# What's new?

* List pdf.js as a package, similar to [the install profile](https://github.com/Islandora-Devops/islandora_install_profile_demo/blob/e20dd33b6a09344620c65ab7104cb4a16c4ca55d/composer.json#L54-L65). 
* Require it.

* Does this change add any new dependencies? **yes, it adds one that was missing!**
* Does this change require any other modifications to be made to the repository
 (i.e. Regeneration activity, etc.)? **no**
* Could this change impact execution of existing code? **no**

# How should this be tested?

* If you have a starter site on ISLE before this PR, notice it doesn't include pdf.js
* Spin up a new starter site on ISLE with this PR. Create a PDF and see that it displays in the viewer.

# Documentation Status

* Does this change existing behaviour that's currently documented? ** no**
* Does this change require new pages or sections of documentation? **no**
* Who does this need to be documented for? **n/a**
* Associated documentation pull request(s): ___  or documentation issue ___

# Additional Notes:
Any additional information that you think would be helpful when reviewing this
 PR.

# Interested parties
Tag @nigelgbanks @adam-vessey
